.github/workflows/ci-nix.yml

```diff
@@ -53,6 +53,9 @@ jobs:
       - name: Clippy workspace
         run: nix build -L --extra-experimental-features nix-command --extra-experimental-features flakes .#workspaceClippy
 
+      - name: Run cargo doc
+        run: nix build -L --extra-experimental-features nix-command --extra-experimental-features flakes .#workspaceDoc
```

Comment:
I wonder in how far we could use the result to publish to gh pages if built on master? So we don't need the other job anymore?

Reply:
We could publish these just fine. Just without index page because it's a nightly feature. See https://github.com/fedimint/fedimint/issues/464

Reply:
Ah, ok, then let's leave it like this for now. We might add an overview page to the wiki once it launches on fedimint.org that can link to the different crates directly. But for now I'd like to keep the index.